src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWAMQyi0S1b073eKUTZN###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_7@9
A "NullPointerException" could be thrown; "message" is nullable here.

Reply:
simulating reply on comment

Reply:
simulating auto reply on comment